pyatlan/generator/templates/structs.jinja2

```diff
@@ -11,14 +11,15 @@ from pyatlan.model.enums import (
     BadgeConditionColor,
     SourceCostUnitType,
 )
-from pyatlan.utils import validate_required_fields
+from pyatlan.utils import init_guid, validate_required_fields
 
 {% for struct in struct_defs %}
 class {{struct.name}}(AtlanObject):
     """Description"""
     {%- if struct.name == "BadgeCondition" %}
     @classmethod
     # @validate_arguments()
+    @init_guid
```

Comment:
I don't think there's a GUID on structs — unsure why this would be needed here?

Reply:
Ok. I will remove it, but it does check if the class doesn't have a guid it does nothing.